meta/meta.go

```diff
@@ -733,9 +733,9 @@ func (m *Meta) GetDDLReorgHandle(job *model.Job) (startHandle, endHandle, physic
 		err = errors.Trace(err)
 		return
 	}
-	// endHandle or physicalTableID may be 0, because older version TiDB (without table partition) doesn't store them.
+	// physicalTableID may be 0, because older version TiDB (without table partition) doesn't store them.
 	// update them to table's in this case.
-	if endHandle == 0 || physicalTableID == 0 {
```

Comment:
why ignore `endHandle` here?

Reply:
Because `endHandle` is 0 don't mean doesn't store them before. Maybe the stored `endHandle` is 0.

Reply:
Is there any test to cover here?

Reply:
yes, https://github.com/pingcap/tidb/pull/10475/files#diff-e259172e2953a3597b096f11d3c8d3daR1289 this test do this job.